packages/extension-horizontal-rule/src/horizontal-rule.ts

```diff
@@ -38,10 +38,10 @@ export const HorizontalRule = Node.create<HorizontalRuleOptions>({
   addCommands() {
     return {
       setHorizontalRule:
-        () => ({ chain }) => {
+        options => ({ chain }) => {
```

Comment:
I think this `options` is not typed in the Commands typings as far as I can see? Could you please add Typescript typings for this?